content/800-data-platform/05-data-browser.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Data Browser'
+metaTitle: 'Data Browser'
+metaDescription: 'Data Browser'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Use the **Data Browser** in the Prisma Data Platform to view and edit your application data.
+
+</TopBlock>
+
+## Models (Tables or Collections)
+
+When you first open the **Data Browser**, you can review the tables in your application database which are based on the models defined in the Prisma schema file.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-models-view_zs1ihu.png" alt="Data Browser - Models view"/>
+
+You can select a table and its data opens in a new tab.
+
+### Open and close tables
+
+To open another table, click the **+** button.
+
+To close a table, click the the **x** button in the table tab.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-close-open-table_g7xho7.png" width="375px" alt="Data Browser - Open and close tables"/>
+
+### Filter data
+
+Use the **Filters** menu to filter data in the table by adding conditions.
+
+**Steps**
+
+1. Click **Filters** and select **Add a new filter**.
+2. Configure the filter.
+   1. Select the field by which you want to filter.
+   2. Select a comparison operator.
+      - **equals**
+      - **in**
+      - **notin**
+      - **lt**
+      - **lte**
+      - **gt**
+      - **gte**
+      - **not**
+   3. Type the value you want to use for the filter.<br />
+      **Step result**: The table is filtered immediately after you define the first filter.
+3. To add a new filter, click **Add a new filter** and repeat the steps above.
+4. To remove a filter, click the **x** button on the right.
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-filters-actions_grtsrh.png" width="450px" alt="Data Browser - add remove apply filters"/>
+
+**Result**
+
+* The data in the table is filtered based on the conditions you added.
+* The default **None** value in the **Filters** menu changes to the number of filters you added.
+
+### Show and hide columns
+
+You can select which columns to view or hide by using the **Fields** menu.
+
+**Steps**
+
+1. Click the **Fields** menu.
+2. Select only the fields you want to see and deselect any fields you want to hide.
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-fields-actions_aa89rv.png" width="400px" alt="Data Browser - show and hide columns"/>
+
+**Result**
+
+The table is immediately filtered to exclude the data from any fields you have deselected.
+
+Also, the **Fields** menu shows the number of fields that are currently selected.
+
+### Show and hide rows
+
+You can also select to show or skip a specific number of rows in the table view.
```

Comment:
I think we usually call "rows" in Prisma context "entries".

Reply:
We use "records" quite a bit more in the docs.

Reply:
I've found only 6-7 occurrences of "entries" and they felt interchangeable with "records". Or am I missing something?

Reply:
No, I just don't like "records" that much so keep ignoring our usage 😆

Reply:
Can you take our usage into account for the purposes of this ticket? :) And have a terminology discussion later?

Reply:
@janpio I'll stick to using "records" here and will start a terminology discussion separately.

Reply:
Update: I've started a docs issue to evaluate the switch to using 'entries' here: https://github.com/prisma/docs/issues/3533

It sounds like the widespread use of 'records' is due to the fact that MongoDB was only added this year and we didn't have a good umbrella term before that.

Reply:
Resolving the convo here.